Search: add folders and subforders to search info (#93177)
grafana/grafana: docs/sources/search/_index.md

weight: 80
---

# Search dashboards and folders

> You can search for dashboards and folders by name.

Reviewer: I'd like to be explicit about this.
```suggestion
You can search for dashboards and dashboard folders by name.
```

When you search for dashboards, you can also do it by panel title. Whether you search by name or panel title, the system returns all dashboards available within the Grafana instance, even if you do not have permission to view the contents of the dashboard.

## Search by name

Begin typing any part of the dashboard or folder name in the search bar. The search returns results for any partial string match in real-time, as you type.

The search is:

- Real-time
- _Not_ case sensitive
- Functional across stored _and_ file based dashboards.

{{% admonition type="note" %}}
You can use your keyboard arrow keys to navigate the results and press `Enter` to open the selected dashboard or folder.
{{% /admonition %}}

The following image shows the search results when you search a dashboard by name.

{{< figure src="/static/img/docs/v91/dashboard-features/search-by-dashboard-name.png" width="700px" >}}

The following images shows the search of a folder and the search for a dashboard inside that folder.

> TODO >>>> add screenshots

Author: I'll wait for the text to be approved to add the screenshots

> The following images shows the search of a folder and the search for a dashboard inside that folder.

Reviewer: The page starts out talking about searching for dashboards and folders by name, so maybe there should be three total examples:

- Searching by dashboard name from the Dashboards page
- Searching by folder name from the Dashboards page
- Searching by dashboard name inside a folder

Author: Yes. The first one, '_Searching by dashboard name from the Dashboards page_', was already on the Search page. The following two, '_Searching by folder name from the Dashboards page_' and '_Searching by dashboard name inside a folder_', are the ones that I want to include. 
Do you want me to be more specific? For example:

> The following images show:
1.- Searching by dashboard name from the Dashboards page
|>>> Here the screenshot
2.- Searching by folder name from the Dashboards page
|>>> Screenshot
3.- Searching by dashboard name inside a folder
|>>> Screenshot

Reviewer: Yes, exactly! I think just so it's really clear the two different starting points and what one is searching for.

{{% admonition type="note" %}}

> When searching within a folder, the subfolders will not be return. You need to use the main search page to find them.

Reviewer: ```suggestion
When you search within a folder, its subfolders are not part of the results returned. You need to be on the **Dashboards** page (or the root level) to search for subfolders by name.
```

weight: 80
---

# Search dashboards and folders

You can search for dashboards and dashboard folders by name.

When you search for dashboards, you can also do it by panel title. Whether you search by name or panel title, the system returns all dashboards available within the Grafana instance, even if you do not have permission to view the contents of the dashboard.

## Search by name

Begin typing any part of the dashboard or folder name in the search bar. The search returns results for any partial string match in real-time, as you type.

The search is:

- Real-time
- _Not_ case sensitive
- Functional across stored _and_ file based dashboards and folders.

{{% admonition type="note" %}}
You can use your keyboard arrow keys to navigate the results and press `Enter` to open the selected dashboard or folder.
{{% /admonition %}}

The following images show:

> 1. Searching by dashboard name from the Dashboards page.

Reviewer: ```suggestion
Searching by dashboard name from the **Dashboards** page.
```

{{< figure src="/static/img/docs/v91/dashboard-features/search-by-dashboard-name.png" width="700px" >}}

> 2. Searching by folder name from the Dashboards page.

Reviewer: ```suggestion
Searching by folder name from the **Dashboards** page.
```

{{< figure src="/media/docs/grafana/dashboards/search-folder-by-name.png" width="700px" >}}

> 3. Searching by dashboard name inside a folder.

Reviewer: ```suggestion
Searching by dashboard name inside a folder.
```
